sycl/test-e2e/Graph/graph_exception_global_device_extension.cpp

```diff
@@ -17,24 +16,23 @@ sycl::ext::oneapi::experimental::device_global<int, TestProperties>
 
 enum OperationPath { Explicit, RecordReplay, Shortcut };
 
-template <OperationPath PathKind> void test() {
-  queue Q{{sycl::ext::intel::property::queue::no_immediate_command_list{}}};
+template <OperationPath PathKind, typename T> void test(T Queue) {
```

Comment:
Is there a reason why you prefer to use a template for the queue type here?

Reply:
Just thought the convention would be to make parameters template types too but I'm happy to change it specifically to a `queue` if that is preferred?

Reply:
The `Queue` parameter can only ever be a `queue` type based on it's usage in this function, so feel like using a template is just obfuscating things here. Preference for using `queue Queue` rather than `T Queue`

Reply:
Fixed this in my original commit for this PR now.